File: api/code-samples/javascript/klay/filter/uninstallFilter.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));

    const quantity = '0xd32fd16b6906e67f6e2b65dcf48fc272'

    sdk.klay.uninstallFilter(quantity, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: api/code-samples/python/klay/filter/uninstallFilter.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

quantity = "0xd32fd16b6906e67f6e2b65dcf48fc272"

sdk = OpenSDK(host)
klay_response = sdk.klay.uninstall_filter(quantity)

print(json.loads(klay_response.response.data))
